team-manager/src/main/java/ua/knu/knudev/teammanager/service/AccountProfileService.java

```diff
@@ -74,18 +105,39 @@ private AccountProfile buildAccountProfile(AccountCreationRequest request,
                 .firstName(reqFullName.firstName())
                 .lastName(reqFullName.lastName())
                 .middleName(reqFullName.middleName())
-                .avatar(uploadFilename)
+                .avatarFilename(uploadFilename)
                 .department(department)
                 .specialty(specialty)
+                .registrationDate(LocalDateTime.now())
+                .expertise(request.expertise())
+                .roles(new HashSet<>(authAccount.roles()))
                 .build();
     }
 
     private AccountRegistrationResponse buildRegistrationResponse(AccountProfile savedAccount,
-                                                                  String requestEmail) {
-        AccountProfileDto account = accountProfileMapper.toDto(savedAccount);
+                                                                  String requestEmail,
```

Comment:
Don't need write 1 property on 1 line, write as I told you, it'll be better

Reply:
done